### PR TITLE
removes the fireball wand from drake loot because I don't like it

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

removes the fireball wand from drake loot, sacred flame is now dropped with the lava staff

### Why is this change good for the game?
I don't like the fireball wand
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.
fireball wand is gone from drake loot, sacred flame drop is now alongside the lava staff rather than the fireball wand

### What general grouping does this PR fall under? 
mining

# Changelog
:cl:  
tweak: fireball wand is no longer in drake loot pool, sacred flame book drops with the lava staff
/:cl:

### CODE REVIEW

**Comment by LazennG** [comment]
gaming

**Comment by Takahiru** [comment]
i already had fun with it

**Comment by ynot01** [comment]
> i already had fun with it

real title
